src/components/organize/EventList/index.tsx

```diff
@@ -1,25 +1,34 @@
-import { List } from '@material-ui/core';
 import { useIntl } from 'react-intl';
+import { Card, Divider, List } from '@material-ui/core';
 
 import EventListItem from './EventListItem';
 import { ZetkinEvent } from '../../../types/zetkin';
 
 interface EventListProps {
-    events: ZetkinEvent[];
     hrefBase: string;
+    events: ZetkinEvent[];
 }
 
-const EventList = ({ events, hrefBase }: EventListProps): JSX.Element => {
+const EventList = ({ hrefBase, events }: EventListProps): JSX.Element => {
     const intl = useIntl();
-
     return (
-        <List aria-label={ intl.formatMessage({
-            id: 'pages.organizeCampaigns.events',
-        }) } disablePadding>
-            { events.map(event => (
-                <EventListItem key={ event.id } event={ event } hrefBase={ hrefBase } />
-            )) }
-        </List>
+        <Card>
+            <List aria-label={ intl.formatMessage({
+                id: 'pages.organizeCampaigns.events',
+            }) }>
+                { events.map((event, index) => (
+                    <>
+                        <EventListItem key={ event.id } event={ event } hrefBase={ hrefBase } />
+                        {
+                            // Show divider under all items except last
+                            index !== events.length - 1 && (
+                                <Divider />
+                            )
+                        }
+                    </>
```

Comment:
`EventList` and `TaskList` are missing key props. This is a good use case for React.fragment, you can pass in a key prop to the fragment by explicitly using 
`<React.fragment key={key}> < /React.fragment>`

Reply:
Done!